sources/PTYTextView.m

```diff
@@ -4831,8 +4833,10 @@ - (void)doCommandBySelector:(SEL)aSelector
         // See comment in -keyDown:
         DLog(@"Rerouting doCommandBySelector from %@ to %@", self, gCurrentKeyEventTextView);
         [gCurrentKeyEventTextView doCommandBySelector:aSelector];
+        [self.delegate keyDown:_keydownEvent];
```

Comment:
`-handleEvent:` only handles keyboard event, but its signature accepts all `NSEvent*`. So when we pass mouse event, `-handlEvent:` return `NO`.

As you say, it seems always return `YES` as long as keyboard event.